src/Tests/Spectre.Console.Tests/Unit/Prompts/TextPromptTests.cs

```diff
@@ -410,4 +410,21 @@ public Task Uses_the_specified_choices_style()
         // Then
         return Verifier.Verify(console.Output);
     }
+
+    [Theory]
+    [InlineData("yes")]
+    [InlineData("Yes")]
+    [InlineData("YES")]
+    public async Task Uses_case_insensitive_comparison_when_no_comparer_is_passed(string input)
+    {
+        var console = new TestConsole { EmitAnsiSequences = true, };
+        console.Input.PushTextWithEnter(input);
+
+        var prompt = new TextPrompt<string>("Was the tool helpful?")
+            .AddChoices(["Yes", "Partially", "No"]);
+
+        var result = await console.PromptAsync(prompt);
+
+        Assert.Equal("Yes", result);
```

Comment:
We're usually using verifiers for this, but it's OK since we're only checking that case-insensitive input was accepted as the expected answer.

We do however use `Shouldy` for asserting things, and not xUnit's Assert library.

```suggestion
        // Then
        result.ShouldBe("Yes");
```